Makefile

```diff
@@ -139,6 +139,14 @@ deploy-operator:
 	make deploy
 	kubectl wait --for=condition=Available=True deploy --all -n 'open-feature-operator-system'
 
+deploy-demo:
+	kubectl apply -f config/samples/end-to-end.yaml
+	kubectl wait --for=condition=Available=True deploy --all -n 'open-feature-demo'
+	kubectl port-forward -n open-feature-demo service/open-feature-demo-service 30000:30000
```

Comment:
Is this always strictly necessary? I believe this is already handled if you're using a kind cluster

Reply:
I believe there is the option to have a default exposed port of `30000` when using the `extraPortMappings` configuration in kind but this is not the case by default.  
This also ensures that the one line deployment will work across all clusters (provided there is nothing already bound to the port)